get_dependencies.py

```diff
@@ -3,11 +3,21 @@
 import tarfile
 from zipfile import ZipFile
 import requests
+from rfc6266 import parse_requests_response
+import posixpath
 
 
-def download(url, filename):
+def download(url):
+    response = requests.get(url, stream=True)
+    filename = parse_requests_response(response).filename_unsafe
```

Comment:
You're doing this twice. Do it only once without the try-except (if there is an exception, we want it to propagate to the caller).

Reply:
my code is like
    response = requests.get(url, stream=True)
    try:
        filename = parse_requests_response(response).filename_unsafe
    except Exception:
        print("Couldn\'t get file name for this URL")
         return 
what should I change here?

Reply:
You've done it 👍